text/0000-slatepack.md

```diff
@@ -0,0 +1,384 @@
+
+4. _The difference between synchronous and asynchronous transaction methods is abstracted away from the end user with the SlatePack standard_
+  - `grin-wallet send -d SlatePackAddress 1.337` will first try to send the Grin synchronously via Tor to the `SlatePack Address`
+  - If that fails it will fall back to outputting an armored encrypted `SlatePack Message` string for manual copy and paste transport
+
+5. _Asynchronous transactions are now encrypted by default by knowing the `SlatePack Address` of your counterparty(s)_
+  - If a counterparty is unwilling or unable to provide a `SlatePack Address`, a plain text `SlatePack Message` can still be exchanged
+  - Example `SlatePack Address`: `grin1p4fuklglxqsgg602hu4c4jl4aunu5tynyf4lkg96ezh3jefzpy6swshp5x`
+
+6. _Sending a mobile Grin transaction should be as easy as scanning a simple QR encoded from a bech32 `SlatePack Address`_
+  - Or as easy as pasting the `SlatePack Address` of your counterparty into your wallet for any other device
+  - Or in the worst case, if Tor is not working or blocked for one of the parties, as easy as copying and pasting a couple of `SlatePack Message` strings with a counterparty in an alternative communication channel (email, forum, social media, instant messenger, generic web text box, carrier pigeon etc.)
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+The SlatePack standard defines the three primary components: `SlatePack Address`, `SlatePack Message` and `SlatePack Workflow`.
+
+The `SlatePack Address` is a shareable bech32 encoded ed25519 public key that can be used both to _route synchronous transactions_ and to _encrypt asynchronous transactions_.
+
+The `SlatePack Message` is an easily copy and pastable ascii-armor string that contains an encrypted slate payload by default and is used in asynchronous transactions.
+
+The `SlatePack Workflow` specifies how both of these components interact in a universally adoptable transaction standard for Grin.
+
+## `SlatePack Address`
+
+A `SlatePack Address` is a bech32 encoded ed25519 public key and when shared with other parties is used to represent the ability to receive Grin transactions.
+
+grin-wallet already handles ed25519 keys for the v3 onion addresses in Tor transactions. These keys can be extended to be a general `SlatePack Address` to allow a universal key format for both transport and encryption that is error-checked, QR friendly and easily human identifiable.
+
+  - Existing ed25519 public keys from the wallet are bech32 encoded with `grin` as the `human-readable part` to build a `SlatePack Address`
+
+  - A `SlatePack Address` can be decoded to its ed25519 public key which can then be mapped to an x25519 public key to be used for encryption
+
+Wallets should support the ability for a user to both use a static, reusable `SlatePack Address` or to generate a unique `SlatePack Address` for each transaction if they wish.
```

Comment:
Consider enforcing disposable **sending** addresses by default (i.e. even if address for receiving is static), as it reduces involuntary privacy leakage.